website/screens/components/select/specs/SelectSpecsPage.tsx

```diff
@@ -143,1056 +350,842 @@ const sections = [
+            <DxcParagraph>
+              The component <Code>width</Code> can adopt the following values:
+            </DxcParagraph>
+            <DxcTable>
+              <thead>
+                <tr>
+                  <th>Width</th>
+                  <th>Value</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>
+                    <Code>small</Code>
+                  </td>
+                  <td>240px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>medium</Code> (<em>default</em>)
+                  </td>
+                  <td>360px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>large</Code>
+                  </td>
+                  <td>480px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>fillParent</Code>
+                  </td>
+                  <td>100%</td>
+                </tr>
+              </tbody>
+            </DxcTable>
+          </>
+        ),
+      },
+      {
+        title: "Margin",
+        content: (
+          <>
+            <DxcParagraph>
+              Select <Code>margin</Code> can use the values:
+            </DxcParagraph>
+            <DxcTable>
+              <thead>
+                <tr>
+                  <th>margin</th>
+                  <th>value</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>
+                    <Code>xxsmall</Code>
+                  </td>
+                  <td>6px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>xsmall</Code>
+                  </td>
+                  <td>16px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>small</Code>
+                  </td>
+                  <td>24px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>medium</Code>
+                  </td>
+                  <td>36px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>large</Code>
+                  </td>
+                  <td>48px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>xlarge</Code>
+                  </td>
+                  <td>64px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>xxlarge</Code>
+                  </td>
+                  <td>100px</td>
+                </tr>
+              </tbody>
+            </DxcTable>
+            <DxcParagraph>
+              These values can be applied independently to each side of the
+              component: <Code>top</Code>, <Code>bottom</Code>,{" "}
+              <Code>left</Code> and
+              <Code>right</Code>.
```

Comment:
Please add a blank space between `and` and `right`.

Reply:
Done!